.github/workflows/newsroom-resilience.yml

```diff
@@ -0,0 +1,52 @@
+name: "[Dotcom] Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 6AM, Mondays to Fridays
+    - cron: "0 6 * * MON-FRI"
+
+jobs:
+  # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-outputs-for-a-job
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ steps.main.outputs.repos }}
+    steps:
+      - id: main
+        run: |
+          REPOS=(
+            newsroom-resilience-platform
+            birthdays
+            pressreader
+          )
+          
+          RESULT=""
+          for repo in "${REPOS[@]}"; do
+            RESULT="$RESULT,guardian/$repo"
+          done
+          
+          # remove leading ,
+          RESULT="${RESULT:1}"
+          echo "repos=$RESULT" >> $GITHUB_OUTPUT
+  prnouncer:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      - uses: guardian/actions-prnouncer@main
+        with:
+          google-webhook-url: ${{ secrets.NEWSREL_GOOGLE_WEBHOOK_URL }}
```

Comment:
I can't access the secrets for this repo, so just wanted to check that this is the right variable name? `NEWSREL_GOOGLE_WEBHOOK_URL` (I don't think I've seen 'newsrel' used before for the NR team so just wanted to check that it's not a typo.)

Reply:
I've added the secret to this repo, I used the `NEWSREL` abbreviation to get the length down a bit!